Core/Core/Logging/SpeckleLog.cs

```diff
@@ -247,17 +247,17 @@ public static void OpenCurrentLogFolder()
     {
       Process.Start(_logFolderPath);
     }
-    catch (Exception ex)
+    catch (FileNotFoundException ex)
```

Comment:
Could this potentially be another exception, such as unauthorised access?

Reply:
Good point, perhaps we can make this catch all `IOException` and `UnauthorisedAccessException`
Would be a safer change.

I did check seq for this error template, zero occurrences in the last 3 months.

Reply:
🙌🏼 Let's move it to be `IOException` just for extra safety.